airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/destination/StreamSyncSummary.kt

```diff
@@ -3,17 +3,9 @@
  */
 package io.airbyte.cdk.integrations.destination
 
-import java.util.*
+import io.airbyte.protocol.models.v0.AirbyteStreamStatusTraceMessage.AirbyteStreamStatus
```

Comment:
for later:  we need to clean up this v0 protocol from dependency before we introduce protocol versioning. most places we use classes from v0 and some non-v0